doc/topics/proxyminion/index.rst

```diff
@@ -36,6 +36,21 @@ or more minions.
 See :ref:`Proxyminion Beacon <proxy-minion-beacon>` to help
 with easy configuration and management of ``salt-proxy`` processes.
 
+New in Nitrogen
```

Comment:
Would you agree to have them in both places? People tend to neglected reading one or the other.